stacker/actions/build.py

```diff
@@ -89,6 +89,10 @@ def resolve_parameters(parameters, blueprint, context, provider):
             values = value.split(",")
             for v in values:
                 v = v.strip()
+                # Skip parameters not referencing another stack
```

Comment:
It feels like this logic is in the wrong place - honestly because the original logic was in the wrong place.  We should probably pull the for loop that iterates over each part of the Parameter outside of the logic that checks for `::`.  SOmething like:

```
v_list = []
values = value.split(",")
for v in values:
  ... handle regular values, output reference values, etc ...
```

does that make sense?  It'd make it easier, later, to add other special values other than just the `::` values as well.